action/receipt.go

```diff
@@ -19,6 +21,24 @@ const (
 	FailureReceiptStatus = uint64(0)
```

Comment:
Let's move error code definition to https://github.com/iotexproject/iotex-proto/tree/master/proto/types